src/contracts/BaseParaSwapSellAdapter.sol

```diff
@@ -99,7 +105,7 @@ abstract contract BaseParaSwapSellAdapter is BaseParaSwapAdapter {
       }
     }
     require(
-      amountToSwap == balanceBeforeAssetFrom - assetToSwapFrom.balanceOf(address(this)),
+      balanceBeforeAssetFrom - assetToSwapFrom.balanceOf(address(this)) <= amountToSwap,
```

Comment:
Per our convo with Paraswap team, they do not fail in case there is excess, so I think we can be relaxed here.

Reply:
Our convo with paraswap team was regarding buy adapter where they were relaxing the condition that buy can have some excess. In sell, this condition doesn't makes sense because they guarantee that it is always exact in.